src/backend/graphItemService.ts

```diff
@@ -307,4 +317,30 @@ export class GraphItemService {
       badRows: [[RowErrorMessage.SOURCE_TARGET_NOT_FOUND, noExtremitiesRows]]
     };
   }
+
+  /**
+   * Ensure that the current user has the correct privileges to import nodes and create a report of the nodes imported.
+   *
+   * @param sourceKey
+   * @param rc
+   */
+  private static async assertHasReportNodeAccess(rc: RestClient, sourceKey: string): Promise<void> {
+    // The CSV_PLUGIN node will not be returned by LKE if:
+      // 1. As an admin, you are in strict schema and the category is not declared
+      // 2. As a custom group user, the category is not declared in the schema
+      // 3. As a standard user (Custom group), the category is declared but not available
```

Comment:
the CSV_PLUGIN category is created during the bootstrap and deleted after. it shouldn't be showed in the schema

Reply:
Here I mean that someone intentionally created the category and marked it as hidden